src/Data/Text/Builder/Linear/Dec/Unbounded.hs

```diff
@@ -0,0 +1,332 @@
+module Data.Text.Builder.Linear.Dec.Unbounded (
```

Comment:
As usual, feel free to credit yourself at the top of new / significantly extended modules.